### PR TITLE
Added git branch and commit NuGet packages

### PR DESCRIPTION
**Pull request by Haplois**
## Description
Added git branch and commit into NuGet packages.

### CODE REVIEW

File: scripts/build.ps1 [hunk 1]
```diff
@@ -56,6 +56,29 @@ Param(
...
+$TPB_BRANCH = "LOCALBRANCH"
+$TPB_COMMIT = "LOCALBUILD"
+
+try {
+    $TPB_BRANCH = $env:BUILD_SOURCEBRANCH -replace "^refs/heads/"  
+    if ([string]::IsNullOrWhiteSpace($TPB_BRANCH)) { 
+        $TPB_BRANCH = git -C "." rev-parse --abbrev-ref HEAD
+    }
+}
+catch { }
```
Comment (cvpoienaru) [inline scripts/build.ps1:71]: Should we log some error message here ?

File: scripts/build.ps1 [hunk 2]
```diff
@@ -56,6 +56,29 @@ Param(
...
+
+try {
+    $TPB_COMMIT = $env:BUILD_SOURCEVERSION
+    if ([string]::IsNullOrWhiteSpace($TPB_COMMIT)) { 
+        $TPB_COMMIT = git -C "." rev-parse HEAD
+    }
+}
+catch { }
```
Comment (cvpoienaru) [inline scripts/build.ps1:79]: Same as above.

**Comment by Sanan07** [comment]
What is the reason to add this info to Nuget packages?

File: scripts/build.ps1 (hunk 1 above)
Comment (Haplois) [reply scripts/build.ps1:71]: It's not important; an error here should only happen locally if git is not installed already, we can safely ignore the error.

**Comment by Haplois** [comment]
> What is the reason to add this info to Nuget packages?

To make debugging easier, especially in symbol issues.

**Review by Sanan07 — APPROVED** [review]

**Review by cvpoienaru — APPROVED** [review]
LGTM

**Pull request merged**